aws-redshift-resourcepolicy/src/main/java/software/amazon/redshift/resourcepolicy/ClientBuilder.java

```diff
@@ -0,0 +1,16 @@
+package software.amazon.redshift.resourcepolicy;
+
+import java.net.URI;
+
+import software.amazon.awssdk.services.redshift.RedshiftClient;
+import software.amazon.cloudformation.LambdaWrapper;
+
+public class ClientBuilder {
+
+  public static RedshiftClient getClient() {
+    return RedshiftClient.builder()
+            .httpClient(LambdaWrapper.HTTP_CLIENT)
+            .endpointOverride(URI.create("https://aws-cookie-monster-devo.amazon.com"))
```

Comment:
Let's remove this and hide internal stuff as much possible, and if we're doing this elsewhere, we should remove it now,